src/utils/global/ambassadorCard/AmbassadorCard.tsx

```diff
@@ -7,6 +9,8 @@ import { classes } from '../../classes'
 import styles from './ambassadorCard.module.scss'
 import moderatorBadge from '../../../assets/mod.png'
 
+import Tooltip from '../../../pages/overlay/components/tooltip/Tooltip'
```

Comment:
I think tooltip should probably be in utils/global, given that we're using it in a global component here